src/main/resources/openapi/events.yaml

```diff
@@ -121,6 +163,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/EventsResponse'
```

Comment:
as there are more than 1 endpoint now, i think this should be renamed to make it clear that its for the Post request, so maybe something like `EventsPostResponse`